kong/plugins/zipkin/schema.lua

```diff
@@ -11,6 +11,8 @@ return {
 					{ sample_ratio = { type = "number",
 					                   default = 0.001,
 					                   between = { 0, 1 } } },
+					{ peer_service_base = { type = "string",
```

Comment:
What do you mean by "base"?

Reply:
Maybe base is not the best word, but its what simply gets populated in the unknown-service-name field in the screenshots of one of the issues: https://user-images.githubusercontent.com/124244/52150794-51f9dc00-2671-11e9-8e6e-00949df2794f.png .
I am not attached to the name by any means. could be called just ```service_name``` or whatever is fancied.